components/shared/ui/footer.tsx

```diff
@@ -10,8 +10,10 @@ export const Footer: FC<Props> = props => {
   const siteCodename = useSiteCodename();
 
   return (
-    <footer className={`w-full grow-0 h-16 px-10 flex flex-auto shrink-0 justify-end items-center ${mainColorBgClass[siteCodename]} ${props.className ?? ""}`}>
-      <p>Ficto Healthtech inc.</p>
+    <footer className={`${mainColorBgClass[siteCodename]} w-screen`}>
+      <div className="flex gap-4 items-center mx-auto max-w-screen-xl md:h-16">
```

Comment:
```suggestion
      <div className="flex items-center mx-auto max-w-screen-xl md:h-16">
```

I think gap is not necessary as there is only one element :)

Reply:
![image](https://github.com/kontent-ai-bot/kontent-ai-new-sample-app/assets/54802833/9dba5fa2-4bd5-4d31-910a-c38f1469c390)

Let's make it a little bit nice on smaller than 'xl' breakpoints probably add padding px-8 (number can be different) and xl:px-0?

Reply:
35c2922: add x padding fopr footer content